pkg/render/namespaces.go

```diff
@@ -108,8 +108,9 @@ func CreateNamespace(name string, provider operatorv1.Provider, pss PodSecurityS
 
 	switch provider {
 	case operatorv1.ProviderOpenShift:
-		ns.Labels["openshift.io/run-level"] = "0"
 		ns.Annotations["openshift.io/node-selector"] = ""
+		ns.Annotations["security.openshift.io/scc.podSecurityLabelSync"] = "false"
```

Comment:
This annotation is removed in [PR #2724](https://github.com/tigera/operator/pull/2724/files#diff-b906749c8befc3a59df41e3a892f3e0a35204726370e3b4ff124bfe123c7dd95L106) so that OpenShift admission controller will render PSS/PSAs from SCCs on Tigera Operator managed namespaces. As Tigera Operator now  renders PSA/PSS properly, we don't need label sync from OpenShift admission controllers.